NohanaImagePicker/NohanaImagePickerController.swift

```diff
@@ -35,6 +35,8 @@ public enum MediaType: Int {
     @objc optional func nohanaImagePicker(_ picker: NohanaImagePickerController, assetListViewController: UICollectionViewController, cell: UICollectionViewCell, indexPath: IndexPath, photoKitAsset: PHAsset) -> UICollectionViewCell
     @objc optional func nohanaImagePicker(_ picker: NohanaImagePickerController, assetDetailListViewController: UICollectionViewController, cell: UICollectionViewCell, indexPath: IndexPath, photoKitAsset: PHAsset) -> UICollectionViewCell
     @objc optional func nohanaImagePicker(_ picker: NohanaImagePickerController, assetDetailListViewController: UICollectionViewController, didChangeAssetDetailPage indexPath: IndexPath, photoKitAsset: PHAsset)
+    @objc optional func nohanaImagePickerDidTapAddPhotoButton(_ picker: NohanaImagePickerController)
+    @objc optional func nohanaImagePickerDidTapAuthorizeAllPhotoButton(_ picker: NohanaImagePickerController)
```

Comment:
We want you to create[ demo delegate function](https://github.com/nohana/NohanaImagePicker/blob/2aa47db73b902085ce5ce3a5068f322df047db9c/Demo/DemoListViewController.swift#L238).

for example.
```example.swift
func nohanaImagePickerDidTapAddPhotoButton(_ picker: NohanaImagePickerController) {
    print("🐷\(#function)")
}
    
func nohanaImagePickerDidTapAuthorizeAllPhotoButton(_ picker: NohanaImagePickerController) {
    print("🐷\(#function)")
}
```